Tests/StreamChatTests/MockEnpointResponses/Channel.json

```diff
@@ -1191,6 +1191,59 @@
       "pinned_by": null
     }
   ],
+  "pinned_messages": [
+      {
+        "id" : "broken-waterfall-5-7aede36b-b89f-4f45-baff-c40c7c1875d9",
+        "reaction_counts" : {
+
+        },
+        "silent" : false,
+        "created_at" : "2020-06-09T08:10:40.800912Z",
+        "reaction_scores" : {
+
+        },
+        "type" : "regular",
+        "latest_reactions" : [
+
+        ],
+        "text" : "sadfadf",
+        "attachments" : [
+
+        ],
+        "own_reactions" : [
+
+        ],
+        "updated_at" : "2020-06-09T08:10:40.800912Z",
+        "reply_count" : 0,
+        "user" : {
+          "id" : "broken-waterfall-5",
+          "banned" : false,
+          "unread_channels" : 0,
+          "extraData" : {
+            "name" : "Tester"
+          },
```

Comment:
Where did you get these data from? This is not the way we save `extraData` :)

Reply:
I only changed data related to Pin information, so this extradata was already in this json 🤔

Reply:
I extracted a message, from another payload, so that payload has the extraData like this, probably it is a old payload. I'm going to remove it from here

Reply:
We have it in a lot of places tho :D Do you want me to remove them all?
![image](https://user-images.githubusercontent.com/12814114/112182529-44a75f00-8bf5-11eb-93fc-6c579e36593e.png)

Reply:
@nuno-vieira if you can remove it please do, it will only cause confusion in the future.